installer/windows/install.bat

```diff
@@ -1,9 +1,12 @@
 @echo off
+if "%1"=="" (
+    echo "missing required msi path argument"
+    exit /B 1
+)
+
 rem uninstall existing storagenode product
-msiexec /x {E97D368F-CB18-45B5-8799-1EBB10728A99}
+echo uninstalling storagenode
+msiexec /x {E97D368F-CB18-45B5-8799-1EBB10728A99} /passive /qb
 
-rem NB: only installs from *first* release directory
-for /d %%d in (release\*) do (
-	msiexec /i %%d\storagenode_windows_amd64.msi /passive /qb /norestart /log %%d\install.log STORJ_WALLET="0x0000000000000000000000000000000000000000" STORJ_EMAIL="user@mail.example" STORJ_PUBLIC_ADDRESS="127.0.0.1:10000"
-	goto :EOF
-)
+echo installing storagenode from %1
+msiexec /i %1 /passive /qb /norestart /log %~dp1install.log STORJ_WALLET="0x0000000000000000000000000000000000000000" STORJ_EMAIL="user@mail.example" STORJ_PUBLIC_ADDRESS="127.0.0.1:10000"
```

Comment:
Out of curiosity, what `%~dp` is?

Reply:
https://stackoverflow.com/questions/112055/what-does-d0-mean-in-a-windows-batch-file